settings.py

```diff
@@ -968,6 +968,8 @@
 DATADOG_API_KEY = None
 DATADOG_APP_KEY = None
 
+TEST_SHOULD_USE_SQL_BACKEND = False
+
```

Comment:
sure, so i'm gonna answer both your questions here. we want to be able to use the postgres backend in tests. but also keep the ability to run couch tests. this setting will allow us to use `override_settings` to choose some tests to use one or the other. we can then create a decorator to run tests on both settings. see simon's comment here: https://github.com/dimagi/commcare-hq/pull/8780#discussion_r42839279 for more context